pages/apis/graphql/cookbooks/agents.md

```diff
@@ -81,3 +81,38 @@ mutation {
   }
 }
 ```
+
+## Stop an agent
+
+First, get the agent ID. Search for the agent in the org where the `search-string` will match agent name and retrieve the agent ID
```

Comment:
Thanks @nsuma8989 for your contribution!

Just a couple of minor suggested changes...

```suggestion
First, get the agent's ID. Search for the agent in the organization where the `search-string` matches the agent name and retrieve the agent's ID.
```